src/app/admin/(Approval)/approved/page.tsx

```diff
@@ -0,0 +1,157 @@
+'use client'
+
+import React from 'react'
+import Filter from '../../../ui/admin/filter'
+import { inter, nunito } from '../../../ui/fonts'
+import Link from 'next/link'
+import Image from 'next/image'
+import { Input, Button } from '@material-tailwind/react'
+
+const tempData = [
+  {
+    name: 'Trương Samule',
+    MSSV: '20127610',
+    Year: '2020',
+    Email: 'tsamule@gmail.com',
+    link: 'https://facebook.com/',
+    isApproved: false,
+    LinkImg: '/demo.jpg',
+  },
+  {
+    name: 'Trần Hồng Minh Phúc',
+    MSSV: '20127610',
+    Year: '2020',
+    Email: 'tsamule@gmail.com',
+    link: 'https://facebook.com/',
+    isApproved: true,
+    LinkImg: '/demo.jpg',
+  },
+  {
+    name: 'Nguyễn Mai Hoàng Quang Huy',
+    MSSV: '20127610',
+    Year: '2020',
+    Email: 'tsamule@gmail.com',
+    link: 'https://facebook.com/',
+    isApproved: false,
+    LinkImg: '/demo.jpg',
+  },
+  {
+    name: 'Trương Samule',
+    MSSV: '20127610',
+    Year: '2020',
+    Email: 'tsamule@gmail.com',
+    Link: 'https://facebook.com/',
+    isApproved: true,
+    LinkImg: '/demo.jpg',
+  },
+]
+
+export default function page() {
```

Comment:
tên Component viết hoa đi ô
```suggestion
export default function Page() {
```